fuse/dl/models/heads/heads_1D.py

```diff
@@ -105,9 +105,6 @@ def forward(self, batch_dict: Dict) -> Dict:
             batch_dict["model.output." + self.head_name] = prediction
         else:
             logits = self.head_module(global_features)  # --> res.shape = [batch_size, 2, 1, 1]
-            if len(logits.shape) > 2:
-                logits = logits.squeeze(dim=3)  # --> res.shape = [batch_size, 2, 1]
```

Comment:
Why? It happens that we don't get to that scenario?

Reply:
It's a copy-paste from 3D head.